comp/core/gui/guiimpl/views/templates/index.tmpl

```diff
@@ -96,5 +96,13 @@
           <input type="button" id="submit_flare" value="Submit"/>
         </form>
     </div>
+    <div id="error" class="page">
+      <div class="error_page">
+        <div id="error_content"></div>
+        <div id="loggedout">
+          {{ .logginInstruction }}
```

Comment:
```suggestion
        <div id="logged_out">
          {{ .loginInstruction }}
```

Reply:
made in 6b75a44bd4062c7ca00dbdf98769f381ca52644d